### PR TITLE
Button docs fix

### PR DESCRIPTION
**Pull request by prateekbh**
**Merge after migrate21**
Fixes Button's and Fab's Documentation

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -41,11 +41,13 @@
...
+    "jsx-to-string": "^1.1.0",
     "node-sass": "^4.5.3",
     "preact": "^8.2.1",
     "preact-cli": "^1.4.1",
     "preact-material-components": "^1.1.7",
     "prettier": "^1.6.1",
     "sass-loader": "^6.0.6"
-  }
+  },
+  "dependencies": {}
```
Comment (filoozom) [inline docs/package.json]: Why does this need to be added?

Comment (prateekbh) [reply docs/package.json]: fixed

**Comment by prateekbh** [comment]
> the sample code is rendered as <e /> instead of <Button />.

@filoozom, didn't get you?

I see this 
screen shot 2017-09-26 at 11 05 27 am

**Comment by filoozom** [comment]
Oh, strange. I'm getting this:
button-docs

**Comment by prateekbh** [comment]
try `npm i` may be...

**Pull request merged**